lib/auth/mailer.ex

```diff
@@ -1,3 +1,3 @@
-defmodule Auth.Mailer do
-  use Swoosh.Mailer, otp_app: :auth
-end
+# defmodule Auth.Mailer do
+#   use Swoosh.Mailer, otp_app: :auth
+# end
```

Comment:
Merci Simon. 🙏

Reply:
It works fine on a fresh `git clone`. I think it might have been a conflict in my dependencies with another branch.
However `swoosh` is still defined in `mix.lock` file. If we want to remove the dependency completely we can run `mix deps.clean --unused --unlock`